support/tarball/tool-bootstrapping/bootstrap.sh

```diff
@@ -80,32 +80,16 @@ function BuildRefPkgs {
     LogMessage "Copy $refPkgSourceDir to $refPkgsDir"
     cp -r $refPkgSourceDir $refPkgsDir
 
-    LogMessage "Copy coreclr-tools to $refPkgsDir/source-built"
-    cp -r $stage1Dir/Tools/source-built/coreclr-tools/ $refPkgsDir/source-built
-
-    LogMessage "Un-tar private source-built artifacts into $refPkgsDir/source-built/"
-    tar -xzf $stage1Dir/bin/x64/Release/Private.SourceBuilt.Artifacts.*.tar.gz --directory $refPkgsDir/source-built/
-
-    LogMessage "Un-tar sdk into $refPkgsDir/.dotnet/"
-    if [ -d "$refPkgsDir/.dotnet/" ]; then
-        rm -rf $refPkgsDir/.dotnet/
+    LogMessage "Un-tar sdk into $stage1SdkInstallDir"
+    if [ -d "$stage1SdkInstallDir" ]; then
+        rm -rf $stage1SdkInstallDir
     fi
-    mkdir $refPkgsDir/.dotnet/
-    tar -xzf $stage1Dir/bin/x64/Release/dotnet-sdk-3*.tar.gz --directory $refPkgsDir/.dotnet/
-
-    LogMessage "Update SDK version in global.json file"
-    sdkVer=`ls $refPkgsDir/.dotnet/sdk`
-    LogMessage "    Sdk version = [$sdkVer]"
-    sed -i "s|\"dotnet\": \".*|\"dotnet\": \"$sdkVer\"|g" $refPkgsDir/global.json
+    mkdir $stage1SdkInstallDir
+    tar -xzf $stage1Dir/bin/x64/Release/dotnet-sdk-3*.tar.gz --directory $stage1SdkInstallDir
 
     LogMessage "Build stage1 ref-pkgs"
     pushd $refPkgsDir
-    ./build.sh
-    popd
-
-    LogMessage "Package built reference packages into new tarball"
-    pushd $refPkgsDir/artifacts/reference-packages
-    tar --numeric-owner -czf $refPkgsDir/artifacts/Private.SourceBuild.ReferencePackages.bootstrap.tar.gz *.nupkg
+    ./build.sh --with-packages $stage1Dir/bin/x64/Release/Private.SourceBuilt.Artifacts.*.tar.gz --with-sdk $stage1SdkInstallDir
```

Comment:
I think this needs to be a directory.

Reply:
No.  I think it's the tarball.  Looking at the help text in sbrp build.sh, it says "\<dir\>", but it actually takes the archive, because it calls `tar xf` on it.  https://github.com/dotnet/source-build-reference-packages/blob/9cc7dad6c4d9302c00803df5dea41c8a7a59e047/build.sh#L83

Reply:
Oh, sorry, got this confused with the source-build version.  We should maybe allow that there as well, I can log an issue.